cookbook/bundles/override.rst

```diff
@@ -68,7 +68,7 @@ in the core FrameworkBundle:
         $container->setParameter('translator.class', 'Acme\HelloBundle\Translation\Translator');
 
 Secondly, if the class is not available as a parameter, you want to make sure the
-class is always overridden when your bundle is used, or you need to modify
+class is always overridden when your bundle is used or if you need to modify
 something beyond just the class name, you should use a compiler pass::
```

Comment:
@WouterJ Actually, I think we can remove this from 2.3. This approach was considered "bad", but since there was no code change (e.g. in 2.5) that actually made it deprecated, I think we should show the "right" way in all versions. What do you think?